toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -297,6 +298,32 @@ func SystemdFormatCorruptionOption(corruptionOption imagecustomizerapi.Corruptio
 	}
 }
 
+func ParseSystemdVerityOptions(options string) (imagecustomizerapi.CorruptionOption, error) {
+	corruptionOption := imagecustomizerapi.CorruptionOptionIoError
+
+	optionValues := strings.Split(options, ",")
```

Comment:
The `systemd.verity_root_options=` or `systemd.verity_usr_options=` can not only take the corruption options, but there are a lot more other options, how will you plan to handle them? For example, in the LG dev branch, we have `systemd.verity_root_options=panic-on-corruption,root-hash-signature=<PATH>`, which will cause error once rebased this commit.

Reply:
We can expand this function to handle other options in the future. In particular, in your change, you'll want to expand this function to also output the value of `root-hash-signature` so that you reapply that setting when reinitialize verity is used.